layouts/index.json

```
@@ -2,11 +2,11 @@
 {{- if ne $page.Type "json" -}}
 {{- if and $index (gt $index 0) -}},{{- end }}
 {
-	"uri": "{{ $page.Permalink }}",
+	"uri": "{{ .Site.Data.Solo.DocsVersion }}{{ $page.RelPermalink }}",
```

doesn't the range function change the current scope of your jinja templates? so this should really be:
```suggestion
	"uri": "{{ $.Site.Data.Solo.DocsVersion }}{{ $page.RelPermalink }}",
```

---

good question, I don't think that applies to this function (whereas it might for an implicit/annonymous range operation) because the urls are resolving correctly, see http://35.232.45.215/gloo/latest/index.json